build.cake

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // TOOLS
 ///////////////////////////////////////////////////////////////////////////////
-#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.1.20160523.144749
+#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.2.20160623.172854-eap5"
```

Comment:
@devlead no, even after I fixed that (I noticed that as well) I was getting this:

![image](https://cloud.githubusercontent.com/assets/1271146/16380835/410c8826-3c71-11e6-850d-902cb647a278.png)